Doc/reference/datamodel.rst

```diff
@@ -1904,6 +1904,8 @@ Attribute lookup speed can be significantly improved as well.
    and *__weakref__* for each instance.
 
 
+.. _note-slots:
```

Comment:
```suggestion
.. _datamodel-note-slots:
```

In general, following at least ad hoc convention (especially in the stdlib docs), I typically "namespace" ref target labels with the page name (or similar) to avoid collisions and make it more clear what they point to, since there may be other notes about `__slots__` other places, and ref targets are a shared namespace across the entire docs (and even beyond, for the many other projects referencing the CPython docs via Intersphinx).